ClientLibrary/Quickstarts/dotnet/MaterializeUserQuickStarts/Program.cs

```diff
@@ -15,6 +15,7 @@ class Program
         static void Main(string[] args)
         {
             AuthenticationContext ctx = GetAuthenticationContext(null);
+            //GUIDs are for Azure DevOps App ID and Visual Studio Client ID and don't need to be changed 
```

Comment:
👍 that's probably a useful clarification. Bikeshedding: the other way would be `const` values with names to give the info on which is which